ndcube/ndcube.py

```diff
@@ -457,8 +460,8 @@ def crop_by_coords(self, lower_corner, interval_widths=None, upper_corner=None):
         lower_pixels = corners_array.min(0)
         upper_pixels = corners_array.max(0)
         # Creating a tuple to crop the data with inputed coordinates
-        item = tuple([slice(int(lower_pixels[i]), int(upper_pixels[i])) for i in range(n_dim)])
-
+        item = tuple([slice(int(lower_pixels[i]), int(upper_pixels[i]))
+                      for i in range(n_dim)])
```

Comment:
Just out of curiosity, why are you breaking up this line and others like it into two?  According to the PEP8 standard, lines of code can be up to 99 characters long.  It doesn't affect the functionality.  It's just a coding style question.

Reply:
I am coding on Atom and I have a line at 70 characters that I have to avoid to exceed. 
Then I am coding with three windows on my screen so I only can see around 75 characters. This is better for me. If you want, I can modify it but I think that for user with a lot of code windows, it is hard to see everything.

Reply:
I think it's general to stick with the PEP8 standard and have lines up to 99 characters.  The 70 line limit and size of your screen is specific to you and wont make it the code more readable for others.  Not every line has to be 99 characters of course.  Split lines where it makes sense.  But if one line is less than 99 characters, don't split it over two.  As I said, it's more readable and keeps the style standardised with the majority of other Python code out there.

Reply:
Ok, I will change that ;)

Reply:
PEP8 says "[Limit all lines to a maximum of 79 characters.](https://www.python.org/dev/peps/pep-0008/#maximum-line-length)" This is a matter of preference, but I think it is a bad idea to go over 80. If the python standard library can make it under 80 characters, I don't see why we shouldn't be able to. The vertical line in Atom should default to 80.

Reply:
The general policy in SunPy core is 99 since they added this to PEP8:

> For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters), provided that comments and docstrings are still wrapped at 72 characters.

I leave the decision up to @DanRyanIrish though.